Runtime error: slice bounds out of range

Hello. I build nwg-drawer for Ubuntu 22.04 in my PPA https://launchpad.net/~ubuntusway-dev/+archive/ubuntu/dev
But program does not run and failed with error:

```
INFO[0000] term: foot                                   
INFO[0000] lang: ru_RU                                  
INFO[0000] Config dir: /home/sunderland93/.config/nwg-drawer 
INFO[0000] Found 0 pinned items                         
INFO[0000] Found 69 desktop files                       
INFO[0000] Skipped 0 duplicates; 26 .desktop entries hidden by "NoDisplay=true" 
INFO[0000] Found 6 associations in /home/sunderland93/.config/nwg-drawer/preferred-apps.json 
INFO[0000] Found 0 search exclusions in /home/sunderland93/.config/nwg-drawer/excluded-dirs 
INFO[0000] Preferring dark theme variants               
INFO[0000] Using style from /home/sunderland93/.config/nwg-drawer/drawer.css 
panic: runtime error: slice bounds out of range [:117] with capacity 72

goroutine 1 [running]:
main.flowBoxButton({{0xc000023858, 0x1d}, {0xc0000174d5, 0x8}, {0xc0002c6411, 0x8}, {0xc0002de448, 0x2f}, {0xc0002a6c6c, 0x7b}, ...})
	github.com/nwg-piotr/nwg-drawer/uicomponents.go:263 +0x4f5
main.setUpAppsFlowBox({0x0, 0x0, 0xc0000103e0?}, {0x0, 0x0})
	github.com/nwg-piotr/nwg-drawer/uicomponents.go:203 +0x298
main.main()
	github.com/nwg-piotr/nwg-drawer/main.go:469 +0x1966
```

--- a/main.go
+++ b/main.go
@@ -21,7 +21,7 @@ import (
 	"github.com/gotk3/gotk3/gtk"
 )
 
-const version = "0.3.0"
+const version = "0.3.1"
 
 var (
 	appDirs         []string

--- a/uicomponents.go
+++ b/uicomponents.go
@@ -248,8 +248,8 @@ func flowBoxButton(entry desktopEntry) *gtk.Button {
 	button.SetImagePosition(gtk.POS_TOP)
 	name := entry.NameLoc
 	if len(name) > 20 {
-		r := []rune(name)
-		name = string(r[:17])
+		r := []rune(name[:17])
+		name = string(r)
 		name = fmt.Sprintf("%s…", name)
 	}
 	button.SetLabel(name)

--- a/uicomponents.go
+++ b/uicomponents.go
@@ -259,8 +259,8 @@ func flowBoxButton(entry desktopEntry) *gtk.Button {
 	terminal := entry.Terminal
 	desc := entry.CommentLoc
 	if len(desc) > 120 {
-		r := []rune(desc)
-		desc = string(r[:117])
+		r := []rune(desc[:117])
+		desc = string(r)
 		desc = fmt.Sprintf("%s…", desc)
 	}
 	button.Connect("button-release-event", func(btn *gtk.Button, e *gdk.Event) bool {

